exporters/js/src/js/printers/unit.ts

```diff
@@ -0,0 +1,13 @@
+export function printUnit(value: number, unit: string): string {
+  let result = value.toString();
+  if (+value !== 0) {
+    if (unit === 'Pixels') {
```

Comment:
Why is it with capital "P"? 
Maybe lowercase `unit` and then compare to "pixels".

Reply:
Supernova returns pixels as Pixels 🤷 It is a good idea to lowercase it and then do the compare, but IMHO it is not really important to do now :) @literat you can improve this (and the same code is in the scss exporter too) if you want to